README.md

```diff
@@ -1,159 +1,93 @@
+cd radworks-governor-upgrade
+```
 
-After creating a new repository from this template, make sure to set any desired [branch protections](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches) on your repo.
+Copy the `.env.template` file and populate it with values
 
-### Coverage
+```bash
+cp .env.template .env
+# Open the .env file and add your values
+```
 
-The [`ci.yml`](.github/workflows/ci.yml) has `coverage` configured by default, and contains comments explaining how to modify the configuration.
-It uses:
-The [lcov] CLI tool to filter out the `test/` and `script/` folders from the coverage report.
+```bash
+forge install
+forge build
+forge test
+```
 
-- The [romeovs/lcov-reporter-action](https://github.com/romeovs/lcov-reporter-action) action to post a detailed coverage report to the PR. Subsequent commits on the same branch will automatically delete stale coverage comments and post new ones.
-- The [zgosalvez/github-actions-report-lcov](https://github.com/zgosalvez/github-actions-report-lcov) action to fail coverage if a minimum coverage threshold is not met.
+### Formatting
 
-Be aware of foundry's current coverage limitations:
+Formatting is done via [scopelint](https://github.com/ScopeLift/scopelint). To install scopelint, run:
 
-- You cannot filter files/folders from `forge` directly, so `lcov` is used to do this.
-- `forge coverage` always runs with the optimizer off and without via-ir, so if you need either of these to compile you will not be able to run coverage.
+```bash
+cargo install scopelint
+```
 
-Remember not to optimize for coverage, but to optimize for [well thought-out tests](https://book.getfoundry.sh/tutorials/best-practices?highlight=coverage#best-practices-1).
+#### Apply formatting
 
-### Slither
+```bash
+scopelint fmt
+```
 
-In [`ci.yml`](.github/workflows/ci.yml), you'll notice Slither is configured as follows:
+#### Check formatting
 
-```yml
-slither-args: --filter-paths "./lib|./test" --exclude naming-convention,solc-version
+```bash
+scopelint check
 ```
 
-This means Slither is not run on the `lib` or `test` folders, and the [`naming-convention`](https://github.com/crytic/slither/wiki/Detector-Documentation#conformance-to-solidity-naming-conventions) and [solc-version](https://github.com/crytic/slither/wiki/Detector-Documentation#incorrect-versions-of-solidity) checks are disabled.
+#### Scopelint spec compatibility
 
-This `slither-args` field is where you can change the Slither configuration for your project, and the defaults above can of course be changed.
+Some tests will not show up when running `scopelint spec` because the methods they are testing are inherited in the `RadworksGovernor`. In order to get an accurate picture of the tests with `scopelint spec` add an explicit `propose` method to the `RadworksGovernor`. It should look like this:
```

Comment:
Currently this is the extent of what i see from `scopelint spec`. Do you see the same thing?
<img width="540" alt="Screenshot 2023-11-23 at 10 52 33 PM" src="https://github.com/ScopeLift/radworks-governor-upgrade/assets/5423313/03d44ba1-aeca-4c0f-aa14-9a1aa639bbb0">

Reply:
cool -- shouldn't it still show more tests though, the ones unrelated to `propose`?  Anyway, don't let this block merge, if it's an issue in the future then we can revise.

Reply:
At this point it the codebase it doesn't look like it should. It finds the tests to show based on the name of the test contracts. Right now the only test contracts with tests are `Propose` and `Constructor`